mrmustard/physics/gaussian_integrals.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains gaussian integral functions and related helper functions.
+"""
+from typing import Sequence, Tuple
+import numpy as np
+
+from mrmustard import math
+from mrmustard.utils.typing import ComplexMatrix, ComplexVector
+
+
+def real_gaussian_integral(
+    Abc: tuple,
+    idx: tuple[int, ...],
+):
+    r"""Computes the Gaussian integral of the exponential of a real quadratic form.
```

Comment:
This will not show in the docs, unless you do the following:
- You see how in `doc >> code >> physics >> utils` there is a `bargmann.rst`? Create a file called `gaussian_integral.rst`
- In `doc >> code >> physics >> bargmann.rst` (that is, the one OUTSIDE OF UTILS), add the line `utils/gaussian_integral`

Reply:
Thanks, sorry always forgot to do this.